README.md

```diff
@@ -18,8 +18,8 @@ compact, particularly when serialized.
 In summary, the particularly interesting characteristics of the t-digest are that it
 
 * has smaller summaries when serialized
-* works on double precision floating point as well as integers.
-* provides part per million accuracy for extreme quantiles and typically <1000 ppm accuracy for middle quantiles
+* works on double-precision floating points as well as integers.
```

Comment:
This usage of floating point as a noun by analogy with integers is interesting. My feeling, however, is that floating point here is an adjective, not a noun (an elision of "floating point number", contrast with "integer number") and thus should not be made plural.